spec/controllers/state_file/questions/nj_retirement_warning_controller_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+RSpec.describe StateFile::Questions::NjRetirementWarningController do
+  describe ".show?" do
+    let (:intake) { create :state_file_nj_intake }
+    before do
+      allow_any_instance_of(Flipper).to receive(:enabled?).with(:show_retirement_ui).and_return(true)
```

Comment:
[pebble] i know it's tedious but we should have the tests for the flag being off too... the price for using flags in production